.gitignore

```diff
@@ -54,3 +54,7 @@ build-iPhoneSimulator/
 _site/
 .sass-cache/
 .jekyll-metadata
+
+# Node
+node_modules
+!node_modules/avalanche-css
```

Comment:
I think that we only use a single file (`@import '../node_modules/avalanche-css/_avalanche.scss';`). Let's version only this one?